scipy/stats/tests/test_discrete_distns.py

```diff
@@ -646,3 +646,23 @@ def test_gh20692(self):
         pmf = dist.pmf(k)
         pmf_k_int32 = dist.pmf(k_int32)
         assert_equal(pmf, pmf_k_int32)
+
+
+class TestPoissonBinomial:
+    def test_pmf(self):
+        # Test pmf against R poisbinom. Other methods tested with generic tests.
```

Comment:
LMK what other things you want to see here (that the generic tests don't cover). Note that this is supposed to be a basic implementation consistent with the quality of others in SciPy; 16-digit accuracy for extreme parameters, etc., is left to enhancement PRs.